RunningOrder/Story/Managers/StoryManager.swift

```diff
@@ -138,6 +131,35 @@ final class StoryManager: ObservableObject {
             }
         }
     }
+
+    var allStories: [Story] {
```

Comment:
you need it at only one place, maybe remove this var and replace the caller with `stories.values.flatMap { $0 }`directly :)

Reply:
it is also called in SearchBarSuggestions+Logic